packages/matchbox/src/components/Page/Page.js

```diff
@@ -1,149 +1,218 @@
-import React, { Component } from 'react';
+import React from 'react';
 import PropTypes from 'prop-types';
-
-import { buttonFrom } from '../Button';
-import { ChevronLeft } from '@sparkpost/matchbox-icons';
+import styled from 'styled-components';
+import { margin } from 'styled-system';
+import { pick } from '@styled-system/props';
+import { createPropTypes } from '@styled-system/prop-types';
+import { ChevronLeft, MoreHoriz } from '@sparkpost/matchbox-icons';
+import { ActionList } from '../ActionList';
+import { Box } from '../Box';
+import { Button } from '../Button';
 import { EmptyState } from '../EmptyState';
-import { UnstyledLink, linkFrom } from '../UnstyledLink';
+import { Popover } from '../Popover';
+import { UnstyledLink } from '../UnstyledLink';
+import { ScreenReaderOnly } from '../ScreenReaderOnly';
 import { filterByVisible } from '../../helpers/array';
 
-import styles from './Page.module.scss';
+const Wrapper = styled('div')`
+  ${margin}
+`;
+
+function Breadcrumb({ content, ...rest }) {
+  return (
+    <UnstyledLink {...rest} fontSize="200" lineHeight="200" fontWeight="medium">
```

Comment:
Looks like these props, font-size and font-weight, get leaked through to the DOM.

<img width="926" alt="Screen Shot 2020-03-19 at 5 09 56 PM" src="https://user-images.githubusercontent.com/1335605/77115327-81e67900-6a04-11ea-912b-c7ebc4cbf3c7.png">

Reply:
Styled components doesn't have a way to omit props – they do have a `shouldForwardProps` utility that should be released soon 🤞